dcpquery/db/__init__.py

```diff
@@ -174,8 +136,126 @@ def create_recursive_process_functions_in_db():
                             AND file_uuid=NEW.file_uuid
                         )
                         DO INSTEAD NOTHING;
-                    """)
-    config.db_session.commit()
+        """
+
+
+class ProcessProcessLink(SQLAlchemyBase):
+    __tablename__ = 'process_join_table'
+    id = Column(Integer, primary_key=True)
+    child_process_uuid = Column(UUID, ForeignKey("processes.process_uuid"))
+    parent_process_uuid = Column(UUID, ForeignKey("processes.process_uuid"))
+    parent_process = relationship(Process, foreign_keys=[parent_process_uuid])
+    child_process = relationship(Process, foreign_keys=[child_process_uuid])
+
+
+class InitDB:
```

Comment:
I don't think this pattern works well semantically. The class should normally represent a type of thing, not an action. If you want to use a class to organize this, consider the following pattern instead:
```
class DCPQueryDBManager:
    def init_db(self, dry_run=True):
        ...

DCPQueryDBManager().init_db()
```

Reply:
👍